src/main/java/com/oltpbenchmark/benchmarks/tpch/generation/AbstractRandomInt.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oltpbenchmark.benchmarks.tpch.generation;
+
+import static com.google.common.base.Preconditions.checkState;
+
+public abstract class AbstractRandomInt
+{
```

Comment:
Make sure the new code matches the same code style as the rest of the repository.